backend/routes/messageRoute.js

```diff
@@ -0,0 +1,45 @@
+const router = require('express').Router();
+
+const messageService = require('../services/messageService');
+
+const SuperError = require('../errors').SuperError;
+const ERR_BAD_REQUEST = require('../errors').ERR_BAD_REQUEST;
+
+
+router.post('/message', async (req, res, next) => {
+    if(isNaN(req.body.studentId) || isNaN(req.body.employerId)|| typeof(req.body.sendDate) != 'string' || (req.body.sender != 'STUDENT' && req.body.sender != 'EMPLOYER') || typeof(req.body.message) != 'string'){
+        res.status(ERR_BAD_REQUEST).send('Your request structure contains some mistakes. Please try again.');
```

Comment:
Maybe it would be cleaner if we make these verifications in the service and throw the error if we find one? I know we said at the meeting that we make the verifications in the router, but I think those are a bit too many and can be moved to "saveNewMessage" in the service, and we can throw an error from there, which will be caught by the already existing catch.

Reply:
ok, I fixed this